Makefile

```diff
@@ -75,10 +67,8 @@ dist/${MODULE}-$(VERSION).tar.gz: $(SOURCES)
 
 ## clean       : clean up all temporary / machine-generated files
 clean: FORCE
-	rm -f ${MODILE}/*.pyc tests/*.pyc
+	git clean -ffdx || true
```

Comment:
No, this will delete too much . `git clean -fdx` is appropriate for `distclean` but not `clean`